src/transformers/models/rag/configuration_rag.py

```diff
@@ -156,6 +157,7 @@ def __init__(
         self.passages_path = passages_path
         self.index_path = index_path
         self.use_dummy_dataset = use_dummy_dataset
+        self.dataset_revision = dataset_revision
```

Comment:
well,  I guess it's good to support in general, even if we are not considering testing.

Reply:
Agreed. Does this mean that the default configuration for RAG doesn't work, in particular for the examples in the [modeling file](https://github.com/huggingface/transformers/blob/df1542581ee89107eea0569ee044fa8797b66ab0/src/transformers/models/rag/modeling_rag.py#L799), or is it just affecting exact values in the tests?

Reply:
Good catch! The linked example won't work, but it's not due to a change in `transformers`.

Regarding
> the default configuration doesn't work

I am not really sure if we can talk about `works or not` - when we use `use_dummy_dataset=True`. (well, it's dummy dataset, I don't really know the context of its role in real life usage)

Reply:
BTW, I pin someone in the dataset team on slack.

Reply:
Yep, agree, we still want this feature.